modal_global_objects/mounts/modal_client_dependencies.py

```diff
@@ -0,0 +1,87 @@
+# Copyright Modal Labs 2025
+import subprocess
+import tempfile
+
+from modal.config import config
+from modal.exception import NotFoundError
+from modal.image import ImageBuilderVersion
+from modal.mount import (
+    PYTHON_STANDALONE_VERSIONS,
+    Mount,
+)
+from modal_proto import api_pb2
+
+REMOTE_PACKAGES_PATH = "/__modal/.client_pkgs"
+REMOTE_SITECUSTOMIZE_PATH = "/pkg/sitecustomize.py"
+
+def create_client_dependencies(
+    builder_version: ImageBuilderVersion,
+    python_version: str,
+    platform: str,
+    arch: str,
+):
+    profile_environment = config.get("environment")
```

Comment:
Do we ever deploy to a non-`main` Environment? Wondering if we could assert here

Reply:
If so we should probs to so for the other publish scripts as well.